opencog/cython/opencog/atom.pyx

```diff
@@ -10,7 +10,7 @@ cdef class Atom(object):
         del self.handle
 
     def value(self):
-        return void_from_cptr(self.handle)
+        return self.handle.value()
```

Comment:
I guess what you mean by uniquely idenfity is,
```python
>>> # continuing from above
>>> n3 = Atom(n2.value(), a)
>>> n2 == n3
True
```
which can't be done with the change I made, it segfaults. I didn't consider this when I made the change. And if the plan is to remove value() from the public api, then I have to remove its usage in the concept-blending code. Closing pr.